packages/common/src/ui/components/buttons/standard/BaseButton.tsx

```diff
@@ -4,30 +4,29 @@ import {
   Button as MuiButton,
   ButtonProps as MuiButtonProps,
   styled,
+  Theme,
 } from '@mui/material';
 
+const translateColor = (theme: Theme, color?: string) => {
```

Comment:
this was a little offroad - the primary button was showing secondary colours on hover and I didn't like it